recipe/meta.yaml

```diff
@@ -11,10 +11,12 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://gitlab.com/cerfacs/{{ name }}/repository/archive.tar.gz?ref={{ tag }}
   sha256: {{ sha256 }}
+  patches:
+    - 0001-Use-SobolIndicesExperiment.patch
```

Comment:
Not sure about the patches. Should we use the latest commit instead? So have this in the url?
`https://gitlab.com/cerfacs/batman/repository/41565216ebb0d0d9f30ba0e4a9efe825f4f350f1/archive.tar.gz`
As a side note, I am trying to find a way to use this commit number to complete the `__init__.py`

Reply:
Stable versions are good :)

Reply:
OK then there is another patch to apply for the seed issue on windows.

Reply:
I already applied https://gitlab.com/cerfacs/batman/commit/80d471243dd582326d676323d44e04fc9cb60659

Reply:
Now we get an issue when reading the setup file on windows with python3 and the module does not even install ???
With python2 the tests fail as some thread fail but we get no error as it is silently ignored in the try/catch of snapshot class...

Reply:
I guess I'll give up here.

Reply:
OK thanks for your tests. But as I said, we never tested this on windows so I prefer to assume it does not work at all. Which apparently seems to be the case.

Thus, I propose to accept this PR without windows support.

Reply:
Ok, I disabled windows